doc/packaging.md

```diff
@@ -1,30 +1,25 @@
 # Packaging `google-cloud-cpp`
 
 <!-- This is an automatically generated file -->
+
```

Comment:
As the comment says, this is automatically generated. Maybe we should fix the script first and then reformat?

Reply:
I'm happy to come back and fix the script after, if you're happy with the opposite order.

Reply:
Fixing the script later SGTM.